src/client.js

```diff
@@ -83,17 +85,16 @@ export class Saturn {
       }
     }
 
-    let origins = options.origins
-    if (!origins || origins.length === 0) {
-      const replacementUrl = options.url ?? options.cdnURL
-      origins = [replacementUrl]
+    let nodes = options.nodes
+    if (!nodes || nodes.length === 0) {
+      const replacementUrl = options.node ?? { url: this.opts.cdnURL }
```

Comment:
```suggestion
      const replacementNode = options.node ?? { url: this.opts.cdnURL }
```

Reply:
fixed this, had to change the following line as well